docs/resources/aws_cloudfront_distribution.md

```diff
@@ -19,43 +19,47 @@ Ensure that an `aws_cloudfront_distribution` exists:
+
+For available SSL/TLS version identifiers, see [OriginSslProtocols](https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_OriginSslProtocols.html) and [AWS::CloudFront::distribution ViewerCertificate](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudfront-distribution-viewercertificate.html) documentation.
 
 ## Properties
 
-|Property                       	     | Description|
+|Property                                   | Description|
 | ---                                        | --- |
 |distribution\_id                            | The identifier for the CloudFront distribution. |
-|viewer\_protocol\_policies                  | An array of viewer protocol policies for all caches in this distribution; valid policy names are `allow-all` (which allows http and https), `https-only` or `redirect-to-https`. |
-|custom\_origin\_ssl\_protocols              | An array containing SSL/TLS protocols allowed by custom origins in this distribution, empty if there are no no custom origins (i.e. only one or more standard S3 bucket origins). Current valid values: `SSLv3`, `TLSv1`, `TLSv1.1`, `TLSv1.2`. |
-|viewer\_certificate\_minimum\_ssl\_protocol | The minimum SSL/TLS protocol version in the Viewer Certificate. Current valid values: `SSLv3`, `TLSv1`, `TLSv1_2016`, `TLSv1.1_2016`, `TLSv1.2_2018`, `TLSv1.2_2019`, `TLSv1.2_2021`. |
+|viewer\_protocol\_policies                  | An array of viewer protocol policies for all caches in this distribution; valid policy names are `allow-all` (which allows HTTP and HTTPS), `https-only` or `redirect-to-https`. |
+|custom\_origin\_ssl\_protocols              | An array containing SSL/TLS protocols allowed by custom origins in this distribution, empty if there are no custom origins (i.e. only one or more standard S3 bucket origins). Current valid values are `SSLv3`, `TLSv1`, `TLSv1.1`, `TLSv1.2`. |
+|custom\_origin\_protocol\_policy            | Custom origin protocol policy. It is a string, one of "http-only", "match-viewer", "https-only"
+|viewer\_certificate\_minimum\_ssl\_protocol | The minimum SSL/TLS protocol version in the Viewer Certificate. Current valid values are `SSLv3`, `TLSv1`, `TLSv1_2016`, `TLSv1.1_2016`, `TLSv1.2_2018`, `TLSv1.2_2019`, `TLSv1.2_2021`. |
 |s3\_origin\_config                          | True if there are any S3 origin configs in the distribution (i.e. standard S3 bucket origins), false otherwise. |
 
 ## Examples
 
 ### Test that a CloudFront distribution has secure protocols configured
 
     describe aws_cloudfront_distribution('DISTRIBUTION_ID') do
-      its('viewer_certificate_minimum_ssl_protocol') { should_not match /SSLv3|TLSv1$|TLSv1_2016/ }
-      its('viewer_protocol_policies') { should_not include 'allow-all' }
+      its('VIEWER_CERTIFICATE_MINIMUM_SSL_PROTOCOL') { should_not match /SSLv3|TLSv1$|TLSv1_2016/ }
+      its('VIEWER_PROTOCOL_POLICIES') { should_not include 'ALLOW-ALL' }
```

Comment:
@dkumaras Could you revert this change ? these are how actual implementation looks like and the upper case version of the attributes is misleading ..